doc/source/user/getting_started.rst

```diff
@@ -125,10 +125,38 @@ Results Database
 ++++++++++++++++
 
 Results are automatically added to a database and persist between invocations of
-Python. For the example above, we may want to look at the last five results to
-see how :math:`k_\text{eff}` varies with the radius. The
-:class:`~watts.Database` class provides a list-like object that contains all
-previously generated :class:`~watts.Results` objects:
+Python. The ``watts`` command-line tool allows you to quickly get a list of
+results:
+
+.. code-block:: console
+
+    $ watts results
+    +-------+--------+--------+--------+----------------------------+
+    | Index | Job ID | Plugin | Name   | Time                       |
+    +-------+--------+--------+--------+----------------------------+
+    | 0     | 1      | MCNP   |        | 2022-06-01 13:21:44.713942 |
```

Comment:
would it be possible to have `Index` starting by 1 - similar to `Job ID`?

Reply:
The index is the same as if you were to actually load the `Database` object from Python (0-based) and index it as a list, so I think it would be awkward to make it 1-based here.

Reply:
Great! Could you update the tables to make it consistent?